test/fixtures/programmatic/websocket-external-auth/core.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.minimal = async function minimal(event, context) {
+  // eslint-disable-next-line no-console
+  console.info('Event type', event.requestContext.eventType);
+  // eslint-disable-next-line no-console
+  if (event.body) console.info('Event body', event.body);
```

Comment:
Let's remove those logs (I assume the do not play any part in testing)

Reply:
It was copied from the other Websocket fixture which had the same, but no they're not needed and I will delete them.